ignite-generator/src/app/index.es

```diff
@@ -42,6 +42,18 @@ const cleanAndroid = (projectFolder) => {
   Shell.cd('../..')
 }
 
+// This step needed as of Ignite 1.11.0 and RN 0.37.0
+// This wasn't necessary before, might need to be removed at some point
+const removeJestTests = (projectFolder) => {
+  Shell.cd(`${projectFolder}/android`)
+  if (/^win/.test(process.platform)) {
+    Shell.exec('rmdir ./__tests__ /s /q')
+  } else {
+    Shell.exec('rm -rf ./__tests__')
+  }
+  Shell.cd('../..')
```

Comment:
looks good.  I'd prefer we simply do something like 
```js
Shell.rm('-rf', `${projectFolder}/__tests__`)
```

2 nitpicks for this:  Firstly, ShellJS remove command is already platform universal, so no need to regex, it'll just work.  Second, I think you're `cd`ing into the wrong folder on line 48.  Copypasta right?